docs/getting_started/building_and_running_an_example.md

```diff
@@ -0,0 +1,78 @@
+# Building
+
+-   docs/guides/BUILDING.md
+-   Bootstrap to begin building
+    -   will install all your tools etc.
+    -   . scripts/bootstrap.sh - run this first, or if builds fail
+    -   . scripts/activate.sh - faster if you’ve already bootstrapped
```

Comment:
Should we bump these in one more level? 🤔

Reply:
removed this file per request above.